x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_list.tsx

```diff
@@ -202,6 +206,35 @@ export const DataStreamList: React.FunctionComponent<RouteComponentProps<MatchPa
               </EuiFlexItem>
             </EuiFlexGroup>
           </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="s">
+              <EuiFlexItem grow={false}>
+                <EuiSwitch
+                  label={i18n.translate(
+                    'xpack.idxMgmt.dataStreamListControls.includeManagedSwitchLabel',
+                    {
+                      defaultMessage: 'Include managed',
```

Comment:
This copy is fine as is, but `managed streams` seems a bit clearer.
```suggestion
                      defaultMessage: 'Include managed streams',
```

Reply:
I think "managed streams" is too vague. I wouldn't know what that means without the context of the discussion here. But I maybe that's OK since the popup will clarify?

Reply:
Yeah. There is also the issue of confusing this with ILM. I'll update the suggest to `Include agent-managed streams`